tsconfig.json

```diff
@@ -24,6 +24,6 @@
     "noFallthroughCasesInSwitch": true
   },
   "include": [
-    "src"
+    "./src/**/*.ts"
```

Comment:
Same...

Reply:
These happened when I ran yarn install after forking the project. It is an auto updated. 
I have reverted it.


![tsconfig-1](https://user-images.githubusercontent.com/10050194/104602436-b30ef480-567b-11eb-9fed-c6fd6811d3fc.png)